internal/image/validate.go

```diff
@@ -109,7 +109,7 @@ func ValidateImage(ctx context.Context, fs afero.Fs, url string, p *policy.Polic
 		return out, nil
 	}
 
-	inputs, err := a.WriteInputFiles(ctx, fs)
+	inputs, err := a.WriteInputFile(ctx, fs)
```

Comment:
Since it went from "Files" to "File", maybe change the var name to match.

```suggestion
	input, err := a.WriteInputFile(ctx, fs)
```

Reply:
oops missed this one. Fixed. Thanks!